config/game_options.txt

```diff
@@ -315,6 +315,8 @@ MINIMAL_ACCESS_THRESHOLD 20
 ## Comment this out this to make security officers spawn in departmental security posts
 SEC_START_BRIG
 
+## Comment this out to make airlocks use a common wire dictionary instead of a different dictionary for each part of the station.
+WIRES_CAN_USE_DICTIONARY_KEYS
```

Comment:
it's not going to get your pr merged, but for the record this is a pretty terrible name--especially with things that are meant to be read by non coders, you want these kinds of things to be self documenting and easy to understand at a glance

Reply:
Naming it "airlock_wires_always_the_same" would be a misnomer because the config option theoretically impacts other kinds of wired objects.

Reply:
WIRES_USE_AREA